anta/device.py

```diff
@@ -307,7 +299,7 @@ def _keys(self) -> tuple[Any, ...]:
         """
         return (self._session.host, self._session.port)
 
-    async def _collect(self, command: AntaCommand) -> None:
+    async def _collect(self, command: AntaCommand) -> None:  # noqa: C901
```

Comment:
whats C901 -> can we add the name of what we are ignoring?

Reply:
function is too complex.
Could not manage to split it, it seems fine to me.